index.html

```diff
@@ -59,7 +63,136 @@ <h1 class="info">Hey There. <br>
+
+        <div class="image-work">  </div>
+
+        <h3 class="title-multi">
+            Multi-Post Stories
+        </h3>
+
+        <p class="works-description">
+            A daily selection of privately personalized 
+                reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text
+                ever since the 1500s, when an unknown
+                printer took a standard dummy text.
+        </p>
+
+        <ul class="button-container">
+            <li class="button1">css</li>
+            <li class="button1">html</li>
+            <li class="button1">bootstrap</li>
+            <li class="button1">Ruby</li>                
+            <div class="button2">See Project</div>                
+        </ul>          
+
+           
+    </div>
+
+    
+
+    <div class="wrapper-cards">
+
+        <div class="card-1">
+            <div class="image-card">  
+    
+                 <h2 class="title-post"> Profesional Art Printing Data </h2>
+                 <h3 class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h3>
+    
+                 <ul class="button-container">
+                    <li class="button3">html</li>
+                    <li class="button3">bootstrap</li>
+                    <li class="button3">Ruby</li>                                                    
+                 </ul>            
+
+                <div class="orange-button">
+                    See Project
+                </div>        
+            </div>        
+        </div>
+
+        <div class="card-2">
+            <div class="image-card">  
+    
+                 <h2 class="title-post"> Profesional Art Printing Data </h2>
+                 <h3 class="supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h3>
+    
+                 <div class="button-container">                         
+                    <button type="button" class="button3">html</button>
+                    <button type="button" class="button3">bootstrap</button>
+                    <button type="button" class="button3">Ruby</button>                
+                </div>                
+                <div class="orange-button">
+                    See Project
+                </div>        
+            </div>        
+        </div>
+    </div>
+
+    <div class="finale">
+        <h2 class="about-me">
+            About me
+        </h2>
+        <p class="finale-description">
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="get-my-resume">
+            Get My Resume
+        </div>                
```

Comment:
Good job here, but please replace your div with a `button` or `a` tag. Because the div tag is not for providing an action.